### ISSUE TITLE
Hiding Descriptions?

### ISSUE THREAD

**Issue by ijxssie** [Darkhax-Minecraft/Enchantment-Descriptions#95]
Is there a way to hide the descriptions at points? Like a toggle keybind that I'm not seeing or something? I'd love to have it only on the enchantment table or books and not the actual armor and tools, but I don't see an option for it. I also looked in the previous questions and didn't really see an answer. If this is a duplicate question, I'm sorry I couldn't find the solution.

**Comment by Zerether** [comment]
Looking for the same. Would be nice if there would be an Option in the config

**Comment by Darkhax** [comment]
The new update 9.0.8 adds several config options to help with this. 

```json
{
  "enableMod": true,
  "onlyDisplayOnBooks": false,
  "onlyDisplayInEnchantingTable": false
}
```

I have no plans to re-add the shift to display descriptions option because that text is often longer than the description tooltip itself. These options are also only available for 1.18.1 and likely won't be backported.

**Comment by ijxssie** [comment]
Thank you for adding this option!!
